src/go/k8s/Makefile

```diff
@@ -76,11 +76,20 @@ docker-push:
 push-to-kind:
 	kind load docker-image ${IMG}
 
+# Execute end to end tests
+e2e-tests: kuttl docker-build
```

Comment:
nit: This seems to run crlfmt in the background ( through the docker-build target).
Should we maybe skip it since this is also ran in another github action?

Reply:
For the convenience of the developer I would like to leave the `fmt` target tight with the `docker-build` target. I will address this issue in this issue in the next push.